main.py

```diff
@@ -60,18 +60,18 @@ class NewListingsAPI(webapp2.RequestHandler):
     def get(self):
         NEW_LISTING_MAX_DAYS = 5
         new_listing_cutoff = datetime.datetime.now() - datetime.timedelta(days=NEW_LISTING_MAX_DAYS)
-        pairs = Pair.all().filter('created > ', new_listing_cutoff).order('-created').fetch(100)
+        pairs = Pair.all().filter('created > ', new_listing_cutoff).order('-created').fetch(500)
         api_response = {
-            'pairs': [p.to_dict() for p in pairs]
+            'pairs': [p.to_dict() for p in filter_new_listings(pairs)]
```

Comment:
so technically there should be none with < 50 because of the `scam_filter` function when fetching right?